src/HttpHeader.h

```diff
@@ -168,6 +169,8 @@ class HttpHeader
     int len;            /**< length when packed, not counting terminating null-byte */
 
 protected:
+    static const size_t UnrestrictedSize = std::numeric_limits<size_t>::max();
```

Comment:
Hm... The maximum is also affected by `len`'s maximum. If it is _trivial_, we can (and should) change `len` type to size_t. I suspect it is not trivial. In that case, we should change this definition to use the _minimum_ of two values: maximum `decltype(len)` value and maximum `size_t` value, right? If C++ standards guarantee that `int` is never larger than `size_t`, then just using `decltype(len)` is probably fine -- `len` should eventually become size_t.

This change will affect `auto` in another change request.

This change should actually happen in the adjusted maxLen() implementation. See the change request for that method.

Reply:
This is not needed since I removed maxLen().